modules/tensor_mechanics/test/tests/dynamics/wave_1D/tests

```diff
@@ -34,6 +34,31 @@
                    " in a linear elastic material with both Rayleigh damping and"
                    " numerical damping resulting from Hilber-Hughes-Taylor (HHT) time integration."
   [../]
+  [./rayleigh_hht_ad]
+    type = 'Exodiff'
+    input = 'wave_rayleigh_hht_AD.i'
+    exodiff = 'wave_rayleigh_hht_out.e'
+    abs_zero = 1e-09
+    compiler = 'GCC CLANG'
+
+    requirement = "The mechanics system shall correctly predict 1D wave propagation"
+                   " in a linear elastic material with both Rayleigh damping and"
+                   " numerical damping resulting from Hilber-Hughes-Taylor (HHT)"
+                   " time integration when automatic differentiation is used."
+    prereq = "rayleigh_hht"
+  [../]
+  [./rayleigh_hht_ad_jac]
+    type = 'PetscJacobianTester'
+    input = 'wave_rayleigh_hht_AD.i'
+    compiler = 'GCC CLANG'
+
+    requirement = "The mechanics system shall correctly compute the Jacobian"
+                 " for 1D wave propagation in a linear elastic material with"
+                 " both Rayleigh damping and numerical damping resulting from"
+                 " Hilber-Hughes-Taylor (HHT) time integration when automatic"
+                 " differentiation is used."
+    prereq = "rayleigh_hht_ad"
```

Comment:
The pattern in many tests now is to suppress outputs in the jacobian test. Then you should be able to run it concurrently with the exidiff test.

Reply:
We actually explicit set `Outputs/exodus=false` by default with the `PetscJacobianTester` now. You can set `turn_off_exodus_output=True` in the test spec if you do want the exodus output. This way of turning off exodus output from the command line would not save you if you have an exodus sub-block in the Outputs block, however.